### ISSUE TITLE
Fail when bc is not found

### ISSUE THREAD

**Issue by snomos** [giellalt/template-lang-und#3]
At least with `--enable-spellers`.

Presently it just returns `no`, which is not nice - gives surprising results during compilation.

**Comment by flammie** [comment]
I usually use `false` instead of no for the third param of AC_PROG_PATH, it still breaks compilation but slightly more graceful. I have updated the m4 but I do not know how to test it on mac since:

```
$  sudo mv /usr/bin/bc /usr/bin/bc.old
mv: rename /usr/bin/bc to /usr/bin/bc.old: Operation not permitted
```
there's probably some way to get around this...

**Comment by snomos** [comment]
Sorry, no good way around that one - Apple has really locked down their system very hard. Must be tested on linux or with the default GitHub virtual machine for CI.

**Comment by snomos** [comment]
Fixed in 1ff92c3d6d8f66ef65bae7dc995978c72c997670.
